apps/smart-contracts/core/test/AllowedCallers.test.ts

```diff
@@ -8,32 +8,72 @@ import { AllowedCallers } from '../typechain'
 chai.use(smock.matchers)
 
 describe('=> AllowedCallers', () => {
-  let restrictedCallers: AllowedCallers
+  let allowedCallers: AllowedCallers
   let deployer: SignerWithAddress
   let user: SignerWithAddress
+  let user2: SignerWithAddress
 
   beforeEach(async () => {
-    ;[deployer, user] = await ethers.getSigners()
-    restrictedCallers = await allowedCallersFixture()
+    ;[deployer, user, user2] = await ethers.getSigners()
+    allowedCallers = await allowedCallersFixture()
   })
 
   describe('initial state', () => {
-    it('has no allowed accounts', async () => {})
+    it('has no allowed accounts', async () => {
+      expect(await allowedCallers.isCallerAllowed(deployer.address)).to.be.false
+    })
   })
 
   describe('# setAllowedCallers', () => {
-    it('reverts if callers.length != allowed.length', async () => {})
+    it('reverts when callers.length != allowed.length', async () => {
+      await expect(
+        allowedCallers.setAllowedCallers([user.address], [true, false])
+      ).to.be.revertedWith('_callers.length != _allowed.length')
+    })
 
-    it('sets a single account as allowed caller', async () => {})
+    it('sets a single account as allowed caller', async () => {
+      await allowedCallers.setAllowedCallers([user.address], [true])
+      expect(await allowedCallers.isCallerAllowed(user.address)).to.be.true
```

Comment:
```suggestion
      await allowedCallers.setAllowedCallers([user.address], [true])
      
      expect(await allowedCallers.isCallerAllowed(user.address)).to.be.true
```
New line in-between **act** and **assert**. Please do this for testcases below.